experimenter/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx

```diff
@@ -37,18 +37,17 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
               <td data-testid="experiment-channel" className="border-top-0">
                 {displayConfigLabelOrNotSet(experiment.channel, channels)}
               </td>
-
-              {experiment.targetingConfigSlug && (
-                <>
-                  <th className="border-top-0">Advanced Targeting</th>
-                  <td data-testid="experiment-target" className="border-top-0">
-                    {displayConfigLabelOrNotSet(
-                      experiment.targetingConfigSlug,
-                      targetingConfigs,
-                    )}
-                  </td>
-                </>
-              )}
+              <th className="border-top-0">First Run Release Date</th>
+              <td
+                data-testid="experiment-release-date"
+                className="border-top-0"
+              >
+                {experiment.proposedReleaseDate ? (
+                  experiment.proposedReleaseDate
+                ) : (
+                  <NotSet />
+                )}
+              </td>
```

Comment:
Do we want to hide this for non-first-run experiments?

Reply:
This is an open question for now, I'm gonna followup with Shell about that

Reply:
+1 I think we should hide it, as it can confuses the user they haven't set it release date even if its not applicable to them